### PR TITLE
V0.8 new features

### PR DESCRIPTION
**Pull request by loicmathieu**
Maybe wait for the release to merge it, but review can already be done

### CODE REVIEW

File: docs/developer-guide/cicd/helpers/README.md [hunk 1]
```diff
@@ -47,17 +38,41 @@ namespace: io.kestra.tests
...
+
+## Validate the flow to be uploaded to the server
+
+There is a convenient command on the Kestra executable that allows validation of the current flow:
+
+```bash
+./kestra flow validate --local path-to-your-flow.yaml 
+```
+::: tip
+If your flow uses a helper function, flow validation must be done locally as the flow cannot be expanded on the webserver.
```
Comment (Skraye) [inline docs/developer-guide/cicd/helpers/README.md:68]: Precise that local kestra must have plugins used in the flow to validate it.

File: docs/developer-guide/tasks/README.md [hunk 2]
```diff
@@ -148,6 +148,39 @@ tasks:
     <a class="btn btn-primary" href="/plugins/core/tasks/flows/io.kestra.core.tasks.flows.Switch">Switch Task documentation</a>
 </div>
 
+### If
+
+This task processes a set of tasks conditionally depending on a condition. 
+The condition must coerce to a boolean. Boolean coercion allows 0, -0, null and '' to coerce to false,  all other values to coerce to true.
```
Comment (Skraye) [inline docs/developer-guide/tasks/README.md:154]: Is it possible for a value to be undefined?

File: docs/developer-guide/tasks/README.md [hunk 3]
```diff
@@ -261,6 +294,66 @@ tasks:
...
+    cases:
+      case1:
+        - id: case1
+          type: io.kestra.core.tasks.log.Log
+          message: Case 1
+      case2:
+        - id: case2
+          type: io.kestra.core.tasks.log.Log
+          message: Case 2
+      notexist:
```
Comment (Skraye) [inline docs/developer-guide/tasks/README.md:325]: This example is wrong, if you don't have a default case and don't handle the case, the flow will fail anyway.
To make sense, you should have a default case. That way, you may decide to fail on a specific value and just let any other value pass.

File: docs/developer-guide/tasks/README.md (hunk 2 above)
Comment (loicmathieu) [reply docs/developer-guide/tasks/README.md:154]: It's Java so undefined didn't exist and will be null

File: docs/developer-guide/cicd/helpers/README.md (hunk 1 above)
Comment (loicmathieu) [reply docs/developer-guide/cicd/helpers/README.md]: Done

File: docs/developer-guide/tasks/README.md (hunk 3 above)
Comment (loicmathieu) [reply docs/developer-guide/tasks/README.md:325]: Well, it's not wrong but it's maybe too simplified.
I added a default, I'l also send a PR to Kestra as the same example is on the task documentation

**Review by Skraye — COMMENTED** [review]
LGTM

**Pull request merged**